vmdb/spec/models/ems_refresh/refreshers/openstack_refresher_rhos_havana_spec.rb

```diff
@@ -69,7 +69,7 @@ def assert_table_counts
     SystemService.count.should       == 0
 
     Relationship.count.should        == 10
-    MiqQueue.count.should            == 17
+    MiqQueue.count.should            == 18
```

Comment:
Same as above @Ladas @blomquisg, this seems very brittle.  I don't think the intention was to assert 18 miq_queue rows, we must really want to know the types of messages found in the queue.